src/index.js

```diff
@@ -4,7 +4,8 @@ import './style.css';
 import createLi from './modules/create-li.js';
 import addTaskEvent from './modules/add-task.js';
 import getData from './modules/get-data.js';
+import clearCompleted from './modules/clear-completed.js';
```

Comment:
- [ ] Greate work, you can create a new file for the methods that check if the status is completed or not (completed: true/false). please create a new file for these functionalities and import it as a module